include/helib/keys.h

```diff
@@ -326,6 +326,9 @@ class SecKey : public PubKey
   friend std::istream& operator>>(std::istream& str, SecKey& sk);
   friend void ::helib::writeSecKeyBinary(std::ostream& str, const SecKey& sk);
   friend void ::helib::readSecKeyBinary(std::istream& str, SecKey& sk);
+
+  // This just writes the derived part, not including the public key
```

Comment:
Add a similar method for binary serialization
```suggestion
  // TODO: Add a similar method for binary serialization
  // This just writes the derived part, not including the public key
```